src/AWS.Deploy.CLI/Utilities/CommandLineWrapper.cs

```diff
@@ -114,6 +121,27 @@ public class CommandLineWrapper : ICommandLineWrapper
             }
         }
 
+        private static string BuildAWSExecutionEnvValue(string awsExecutionEnv, ProcessStartInfo processStartInfo)
+        {
+            var awsExecutionEnvBuilder = new StringBuilder();
+            if (processStartInfo.EnvironmentVariables.ContainsKey(EnvironmentVariableKeys.AWS_EXECUTION_ENV)
+                && !string.IsNullOrEmpty(processStartInfo.EnvironmentVariables[EnvironmentVariableKeys.AWS_EXECUTION_ENV]))
+            {
+                awsExecutionEnvBuilder.Append(processStartInfo.EnvironmentVariables[EnvironmentVariableKeys.AWS_EXECUTION_ENV]);
```

Comment:
do you need some kind of separator?  Space character or semi colan perhaps?

Reply:
we separate using _, check the description, it has example.

Reply:
Do you need a `.Append("_")`?

Reply:
We don't want _ at the beginning. StringBuilder takes the existing value from env var, and then add _ if there is an already specified AWS_EXECUTION_ENV.

This append is basically taking the existing value, there is no need of append.